src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -162,6 +146,30 @@ private IEventReferenceExpression CreateBoundEventAccessOperation(BoundEventAssi
                  invokedAsExtensionMethod: invokedAsExtensionMethod);
         }
 
+        private IInvalidExpression CreateInvalidExpressionForHasArgumentsExpression(BoundNode ReceiverOpt, ImmutableArray<BoundExpression> arguments, BoundExpression AdditionalNodeOpt, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit)
```

Comment:
Parameter names should begin with lowercase letter: `receiverOpt`, `additionalNodeOpt`.